src/main/java/reposense/git/GitRevList.java

```diff
@@ -98,4 +100,24 @@ private static String getAllCommitHashSince(String root, String branchName, Stri
             return "";
         }
     }
+
+    /**
+     * Returns a list of commit hashes for the root commits in the tree.
+     */
+    public static List<String> getRootCommits(String root) {
+        String revListCommand = "git rev-list --max-parents=0 HEAD";
+        Path rootPath = Paths.get(root);
+        String output = runCommand(rootPath, revListCommand);
+        return Arrays.asList(output.split("\n"));
+    }
+
+    /**
+     * Returns true if the repository is empty.
+     */
+    public static boolean getIsEmptyRepo(String root) {
```

Comment:
Doesn't sound like a boolean returning function with the `get`.  Maybe `check` or `is` will make it sound better.

Reply:
I have changed it to `checkIsEmptyRepo`